targets/unity-v2/source/Shared/Internal/PlayFabHttp/PlayFabWebRequest.cs

```diff
@@ -125,13 +125,21 @@ public void SimpleGetCall(string fullUrl, Action<byte[]> successCallback, Action
             newThread.Start();
         }
 
-        public void SimplePutCall(string fullUrl, byte[] payload, Action successCallback, Action<string> errorCallback)
+        public void SimplePutCall(string fullUrl, byte[] payload, Action<byte[]> successCallback, Action<string> errorCallback)
         {
             // This needs to be improved to use a decent thread-pool, but it can be improved invisibly later
-            var newThread = new Thread(() => SimpleHttpsWorker("PUT", fullUrl, payload, (result) => { successCallback(); }, errorCallback));
+            var newThread = new Thread(() => SimpleHttpsWorker("PUT", fullUrl, payload, (result) => { successCallback(result); }, errorCallback));
```

Comment:
You no longer need to wrap successCallback in a lambda, its signature matches that parameter.
(fixing this will clean up a memory allocation, and avoid general lambda context issues)